src/box/iproto.cc

```diff
@@ -203,6 +203,18 @@ struct iproto_thread {
 	struct evio_service binary;
 	/** Requests count currently pending in stream queue. */
 	size_t requests_in_stream_queue;
+	/** List of all connections. */
+	struct rlist connections;
+	/**
+	 * Number of connections that marked to be dropped but not
+	 * yet dropped.
+	 */
+	size_t drop_count;
```

Comment:
Nit: Better call `drop_pending_count`.

Reply:
Renamed to `drop_pending_connection_count` as discussed f2f.

<details>
<summary>diff</summary>

```diff
diff --git a/src/box/iproto.cc b/src/box/iproto.cc
index 8df0d36f2..7b99ece99 100644
--- a/src/box/iproto.cc
+++ b/src/box/iproto.cc
@@ -205,11 +205,8 @@ struct iproto_thread {
 	size_t requests_in_stream_queue;
 	/** List of all connections. */
 	struct rlist connections;
-	/**
-	 * Number of connections that marked to be dropped but not
-	 * yet dropped.
-	 */
-	size_t drop_count;
+	/** Number of connections that pending drop. */
+	size_t drop_pending_connection_count;
 	/**
 	 * Message used to notify TX thread that all connections marked
 	 * to de dropped are dropped.
@@ -1564,8 +1561,8 @@ iproto_connection_delete(struct iproto_connection *con)
 	if (con->drop) {
 		struct iproto_thread *iproto_thread = con->iproto_thread;
 
-		assert(iproto_thread->drop_count > 0);
-		if (--iproto_thread->drop_count == 0)
+		assert(iproto_thread->drop_pending_connection_count > 0);
+		if (--iproto_thread->drop_pending_connection_count == 0)
 			iproto_send_drop_finished(iproto_thread);
 	}
 	mempool_free(&con->iproto_thread->iproto_connection_pool, con);
@@ -3497,14 +3494,14 @@ iproto_do_cfg_f(struct cbus_call_msg *m)
 			if (!con->is_in_replication)
 				iproto_connection_close(con);
 			con->drop = true;
-			iproto_thread->drop_count++;
+			iproto_thread->drop_pending_connection_count++;
 			if (con->state != IPROTO_CONNECTION_DESTROYED) {
 				cmsg_init(&con->cancel_msg, cancel_route);
 				cpipe_push(&iproto_thread->tx_pipe,
 					   &con->cancel_msg);
 			}
 		}
-		if (iproto_thread->drop_count == 0)
+		if (iproto_thread->drop_pending_connection_count == 0)
 			iproto_send_drop_finished(iproto_thread);
 		break;
 	}

```

</details>